src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -106,9 +117,25 @@ public static void ComputeActiveProcessList(IReadOnlyList<IPositionedOrganelle>
                     continue;
 
                 // If not found, then create a new result
-                result.Add(new TweakedProcess(process.Process, process.Rate));
+                var newProcess = new TweakedProcess(processKey, process.Rate);
+                newProcess.Marked = true;
+                result.Add(newProcess);
             }
         }
+
+        int writeIndex = 0;
```

Comment:
My last comment. This block of code could do with a comment saying this removes the unmarked processes to not keep processes around that don't exist anymore, and also for efficiency this handles unmarking processes that are kept.